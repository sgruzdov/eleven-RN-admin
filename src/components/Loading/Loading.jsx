import React from 'react'
import LinearProgress from '@material-ui/core/LinearProgress'

import * as styles from './Loading.module.scss'

const Loading = () => {
    return (
        <div className={styles.loading}>
            <LinearProgress />
        </div>
    )
}

export default Loading
