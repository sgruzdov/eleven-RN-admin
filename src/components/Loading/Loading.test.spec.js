import Loading from './Loading'

describe('Loading component', () => {
    let component

    beforeEach(() => {
        component = setUp(Loading)
    })

    it('should render Loading component', () => {
        expect(component).toMatchSnapshot()
    })
})
