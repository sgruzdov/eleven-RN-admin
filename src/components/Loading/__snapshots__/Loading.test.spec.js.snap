// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component should render Loading component 1`] = `
<div>
  <WithStyles(ForwardRef(LinearProgress)) />
</div>
`;
